finalPrep/modZeroHero.js

```diff
@@ -1,50 +1,70 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = ("Quad");
+var specialAbility = ("flight");
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
+var greeting = ("Greetings, it is I " + heroName + ".");
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var catchphrase = `With my power of ${specialAbility}, you too can feel safe.`;
 // Declare two variables - power AND energy - set to integers
-
+var power = 8;
+var energy = 9;
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
+var fullPower = (power * 500);
 //   fullEnergy should add 150 to your current energy
-
+var fullEnergy = (energy + 150);
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
-
+var isHuman = true;
+var identityConcealed = true;
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
+var archEnemies = ["ATak", "BTak", "CTak"];
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var sidekicks = ["Real", "Steel", "Veal"];
 // Print the first sidekick to your console
-
+var first = sidekicks[0];
+console.log(first);
 // Print the last archEnemy to the console
-
+var last = archEnemies[2];
```

Comment:
Nice use of bracket notation. A next challenge would be to think about how to print the last element no matter how many elements are in there.